dependency-check-suppressions.xml

```diff
@@ -12,4 +12,8 @@
 		<cve>CVE-2018-1258</cve>
 	</suppress>
 
+	<suppress>
+		<cve>CVE-2018-1258</cve>
+	</suppress>
+
```

Comment:
Looks like this is already suppressed a few lines above, let's change the `until` date to add another month instead!
```suggestion
```